task3/update/index.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" href="logo.png">
+    <title> Ribbitter | Feed </title>
+</head>
+<body>
+<header>
+    <nav class="container">
+        <a class="logo" href="index.html">
+            <img src="logo.png" alt="A VERY MUCH GREEN FROG">
+            <span class="logotext"><h1>Ribbitter</h1></span>
+        </a>
+        <a class="write" href="">
+            <img src="edit.png" alt="Write new post">
+        </a>
+        <a class="exit" href="">
+            <img src="exit.png" alt="exit">
+        </a>
+        <a class="user" href="">
+            <span class="yourname">YourName</span>
+            <img src="you.png" alt="your photo">
+        </a>
+    </nav>
+</header>
+<aside>
+    <h2 class="prefs">Your content preferences:</h2>
+    <form id="filterform">
```

Comment:
if div is a basic block for layout (w/o any meanings and side effects by default)
the form element is a container that first of all brings us behavioral effects.
form is a container for actionable elements (like inputs) that relates to some action.
So in our situation it is the whole filter, it is the add/edit form for tweet or it can be a search input (if we have a separate action for it)